kong.conf.default

```diff
@@ -926,6 +926,13 @@
                                         # connection may be kept open
                                         # indefinitely.
 
+#enable_debug_header = off  # Enable the `kong-debug` header function
```

Comment:
Minor naming thing - other boolean flags in the config are not called `enable_*` - for example `real_ip_recursive = off` is not called `enable_real_ip_recursive = off`.

Is there a reason to not doing this?

```suggestion
#debug_header = off  # Enable the `kong-debug` header function
```

Reply:
as you wish